front/app/(visualiser)/interpeller/[siren]/step1/page.tsx

```diff
@@ -0,0 +1,62 @@
+import Link from 'next/link';
+
+import BadgeCommunity from '@/components/Communities/BadgeCommunity';
+import BudgetGlobal from '@/components/Communities/BudgetGlobal';
+import MiniFicheCommunity from '@/components/Communities/MiniFicheCommunity';
+import RankingCommunity from '@/components/Communities/RankingCommunity';
+import ButtonBackAndForth from '@/components/Interpellate/ButtonBackAndForth';
+import { TransparencyScoreBar } from '@/components/TransparencyScore/TransparencyScore';
+import { TransparencyScore } from '@/components/TransparencyScore/constants';
+import { MoveRight } from 'lucide-react';
+
+export default async function InterpellateStep1({
+  params,
+}: {
+  params: Promise<{ siren: string }>;
+}) {
+  const { siren } = await params;
+  const buttonClassName = `buttonVariants({
+            variant: 'outline',
+            className: 'min-w-[200] bg-black text-white',
+          })`;
+  // TODO - retrieve scores
+  const scores = { subventions: TransparencyScore.E, marchesPublics: TransparencyScore.B };
+  const trends = { subventions: 1, marchesPublics: 0.01 };
+
+  return (
+    <section>
+      <h2 className='mb-12 mt-6 text-center text-2xl font-bold'>Collectivité sélectionnée</h2>
+      <article className='px-8 py-12 outline'>
+        <div className='flex justify-between'>
+          <BadgeCommunity name='exemplaire' />
+          <BudgetGlobal communityName='Nantes' />
+        </div>
+        <div className='mt-10 flex flex-col justify-between md:flex-row'>
+          <MiniFicheCommunity communitySiren={siren} />
+          <div className='min-w-1/4 md:scale-[0.85]'>
+            <h3 className='pl-5 font-bold'>Marchés publics</h3>
+            <TransparencyScoreBar score={scores.marchesPublics} />
+          </div>
+          <div className='min-w-1/4 md:scale-[0.85]'>
+            <h3 className='pl-5 font-bold'>Subventions</h3>
+            <TransparencyScoreBar score={scores.subventions} />
+          </div>
+        </div>
+        <div className='flex justify-between'>
+          <RankingCommunity communityName='Nantes' />
+          <Link className='flex items-end gap-2 hover:underline' href={`/community/${siren}`}>
+            Voir la fiche de la collectivité <MoveRight />
+          </Link>
+        </div>
+      </article>
+      <div className='my-12 flex justify-center gap-4'>
+        <ButtonBackAndForth linkto='/interpeller' direction='back'>
+          Revenir
+        </ButtonBackAndForth>
+        <ButtonBackAndForth linkto={`/interpeller/${siren}/step2`} direction='forth'>
+          Continuer
+        </ButtonBackAndForth>
+      </div>
```

Comment:
Ce que je voulais dire pour les boutons c est de faire deux buttons dans le composant :

```suggestion
      <ProgressButtons
        props1={{ label: 'Revenir', link: '/interpeller' }}
        props2={{ label: 'Continuer', link: `/interpeller/${siren}/step2` }}
      />
```

avec : 

```
import Link from 'next/link';

import { buttonVariants } from '@/components/ui/button';
import { ChevronLeft, ChevronRight } from 'lucide-react';

type ButtonProps = {
  link: string;
  label: string;
};

type ProgressButtonsProps = {
  props1: ButtonProps;
  props2: ButtonProps;
};

export default function ProgressButtons({ props1, props2 }: ProgressButtonsProps) {
  const className = buttonVariants({
    variant: 'outline',
    className: 'min-w-[200] bg-black text-white',
  });

  return (
    <div className='my-12 flex justify-center gap-4'>
      <Link href={props1.link} className={className}>
        <ChevronLeft />
        {props1.label}
      </Link>
      <Link href={props2.link} className={className}>
        {props2.label} <ChevronRight />
      </Link>
    </div>
  );
}
```